api/route.ts

```diff
@@ -204,6 +204,30 @@ export function convertBoulderStringToClassifier(boulderString: string) {
   return new RouteClassifier(rawgrade, RouteType.Boulder);
 }
 
+export function getAllTraverseRouteClassifiers() {
+  return ['Beginner', 'Intermediate', 'Advanced'].map(
+    convertTraverseStringToClassifier
+  );
+}
+
+export function getAllCompRouteClassifiers() {
+  return ['A', 'B', 'C', 'D'].map(convertCompetitionStringToClassifier);
+}
+
+export function getAllBoulderClassifiers() {
+  return ['VB', 'V0', 'V1', 'V2', 'V3', 'V4', 'V5', 'V6'].map(
```

Comment:
can we add V7 just in case tower gets more ambitious? no biggie if not. Also there is no way they will ever set a 5.14 so we are good there